doc/whats_new/v1.1.rst

```diff
@@ -45,6 +45,16 @@ Changelog
   `pos_label` to specify the positive class label.
   :pr:`21032` by :user:`Guillaume Lemaitre <glemaitre>`.
 
+:mod:`sklearn.cluster`
+......................
+
+- |Enhancement| :func:`cluster._spectral` now includes the 'cluster_qr' method
```

Comment:
Please change this to list all impacted functions and classes that are part of the public API instead of private API.